blathering-barnacles/Bangazon-2
# User products

Users can click on a button in the nav bar to take them to a page that displays the products they are selling.  There is also a button to remove an item if a user changes their mind about selling something.

TO TEST
-create an item to sell
-click "Your items for sale" in the nav bar
-check to see if items that ONLY YOU MADE are there
-click cancel sale to delete
